### DISCUSSION TITLE
传送石碑无法被破坏，似乎也没办法重命名他们，很头疼 Can't harvest or rename waystones

### DISCUSSION THREAD

**Discussion by chtarra** [Rad586/Forget-Me-Not#69] [Q&A] [answered]
抱歉打扰您，游戏版本：1.19.2
整合包版本：2.5.0
传送石碑无法被破坏，似乎也没办法重命名他们，很头疼

**Comment by Rad586** [comment, marked as answer]
自然生成的传送石碑无法破坏。
在传送石碑位置可以重命名它。
此外，你还能删除指路石记录点。
Natural generated waystones can't be harvested, you can rename them in the gui of corresponding waystone.
Besides, you can also delete those waypoints.

![image](https://github.com/Rad586/Forget-Me-Not/assets/99306685/803fcb05-f2a3-4b36-b07c-b9729a9e2f77)
![image](https://github.com/Rad586/Forget-Me-Not/assets/99306685/8b99714d-dea2-490c-babe-221691cddb40)
![image](https://github.com/Rad586/Forget-Me-Not/assets/99306685/914e875d-f352-435a-87d9-56eb9ea5be67)
